package.json

```diff
@@ -9,6 +9,7 @@
     "bootstrap": "^4.3.1",
     "bootswatch": "^4.3.1",
     "bowser": "^2.4.0",
+    "ionic": "^5.2.1",
```

Comment:
We probably don't need ionic CLI for this app. We already handle integration with `@ionic/react`